spark/src/main/kotlin/com/adevinta/spark/components/divider/Divider.kt

```diff
@@ -100,14 +125,7 @@ public fun HorizontalDivider(
             color = intent.color(),
         )
 
-        if (!label.isNullOrBlank()) {
-            Text(
-                textAlign = TextAlign.Center,
-                style = SparkTheme.typography.body1,
-                text = label,
-                modifier = Modifier.padding(horizontal = 16.dp),
```

Comment:
Let's move the spacing from the Text to 2 `Spacers` wrapping the label so we can ensure that it'll always be present when a consumer provide a `label`